src/Management/test/EndpointCore.Test/ManagementHostBuilderExtensionsTest.cs

```diff
@@ -504,7 +505,25 @@ public async Task AddAllActuators_IHostBuilder_IStartupFilterFires()
         }
 
         [Fact]
-        public void AddaLLActuators_IHostBuilder()
+        public async Task AddAllActuatorsWithConventions_IHostBuilder_IStartupFilterFires()
```

Comment:
Should there be another test or at least assertion that confirms failure when an unauth'd request hits an auth required actuator?

Reply:
After further review of `MapTestAuthSuccess` this suggestion would likely be excessive